home/home4.1/src/PizzaBuilder.java

```diff
@@ -0,0 +1,19 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class PizzaBuilder {
+    private List<Ingredient> ingredients;
+
+    public PizzaBuilder() {
+        this.ingredients = new ArrayList<>();
+    }
+
+    public PizzaBuilder addIngredient(Ingredient ingredient) {
+        ingredients.add(ingredient);
+        return this;
+    }
+
+    public Pizza build() {
+        return new Pizza(ingredients);
```

Comment:
the same concern for reset of ingredients to ensure that the next pizza can be build